docs/components/morphotagger.rst

```diff
@@ -269,8 +285,8 @@ Chainer
 ^^^^^^^
 
 The ``chainer`` part of the configuration file contains the
-specification of the neural network
-model and supplementary things such as vocabularies. Chainer should be
+specification of the neural network model and supplementary things such as vocabularies.
+Chainer refers to an instance of :class:`~deeppavlov.core.common.chainer.Chainer` and should be
```

Comment:
I think it is also described in "Configuration files" section of the docs. You could write your doc page assuming the user is already familiar with such general things.